pubsub/pulsar/pulsar_test.go

```diff
@@ -37,6 +39,8 @@ func TestParsePulsarMetadata(t *testing.T) {
 	assert.Equal(t, true, meta.DisableBatching)
 	assert.Equal(t, defaultTenant, meta.Tenant)
 	assert.Equal(t, defaultNamespace, meta.Namespace)
+	assert.Equal(t, 100, meta.BatchingMaxSize)
```

Comment:
The tests are failing, the parsing logic does not seem to be working. Can you please take a look @saber-wang ?

Reply:
@shubham1172  Execute make test and throw "make: no rule to make target 'test'. Stop" exception. How should I handle it?

Reply:
That is strange, the Makefile has the `test` command. What environment are on?

Reply:
@shubham1172 
 environment : win11 64bit, PowerShell 7.2.2,Make 4.3
I passed the test with "go test - V"